packaging_automation/common_tool_methods.py

```diff
@@ -171,7 +184,12 @@ def is_merge_commit(commit: Commit):
 @validate_parameters
 def get_version_details(version: is_version(str)) -> Dict[str, str]:
     version_parts = version.split(".")
-    return {"major": version_parts[0], "minor": version_parts[1], "patch": version_parts[2]}
+    release_stage = 'stable'
+    if len(version_parts) == 3:
```

Comment:
asking one more time, can it ever be different than 3 ?

Reply:
It is just an additional check I can remove if you want

Reply:
maybe make it an assert then

Reply:
made it assert